src/library/Uno.Themes.WinUI.Markup/Themes/FloatingActionButton.cs

```diff
@@ -0,0 +1,270 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSurfaceStateOverlayBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabSurfaceStateOverlayBackgroundDisabled", true);
+					}
+				}
+
+				public static class Secondary
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryForeground")]
+						public static ResourceValue<Brush> Default => new("FabSecondaryForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryForegroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabSecondaryForegroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabSecondaryForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabSecondaryForegroundDisabled", true);
+					}
+
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryBackground")]
+						public static ResourceValue<Brush> Default => new("FabSecondaryBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryBackgroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabSecondaryBackgroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabSecondaryBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabSecondaryBackgroundDisabled", true);
+					}
+				}
+
+				public static class SecondaryStateOverlay
+				{
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryStateOverlayBackground")]
+						public static ResourceValue<Brush> Default => new("FabSecondaryStateOverlayBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryStateOverlayBackgroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("FabSecondaryStateOverlayBackgroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryStateOverlayBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FabSecondaryStateOverlayBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryStateOverlayBackgroundFocused")]
+						public static ResourceValue<Brush> Focused => new("FabSecondaryStateOverlayBackgroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FabSecondaryStateOverlayBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FabSecondaryStateOverlayBackgroundDisabled", true);
+					}
+				}
```

Comment:
Should be inside of the `FloatingActionButton.Resources.Secondary` class and renamed to `StateOverlay`

Reply:
Done